DEVELOPING_OPAL/demos/src/main/scala/org/opalj/tac/CloseResources.scala

```diff
@@ -23,7 +23,7 @@ object CloseResources extends ProjectAnalysisApplication {
         params:        Seq[String],
         isInterrupted: () => Boolean
     ): BasicReport = {
-        // implicit val logContext = p.logContext
+        // implicit val logContext: LogContext = p.logContext
```

Comment:
Can't this be removed?

Reply:
Not if one wants to be able to un-comment the rest of the file easily. Now, why there is a demo file where all of the (relevant) code is commented out, that I don't know.